tests/iconGenerator.test.js

```diff
@@ -144,7 +154,7 @@ describe('generating icons', function () {
       var fileExtensions = iconGenerator.buildJsonStructure().files.names.fileExtensions;
 
       files.supported
-        .filter(function (file) { return !file.filename && !file.languages; })
```

Comment:
This removal is due to the decision to include the `language` supported extension in the test.
At first I excluded them but after you clarified it I forgot to change it back.